gen/generate.toml

```diff
@@ -0,0 +1,18 @@
+[general]
+library_name = "libgeos"
+output_file_path = "../lib/libgeos.jl"
```

Comment:
I'd prefer to put this in `src` with the rest of the source code: "../src/libgeos.jl".